sync/twinCards/trello.create_twin.js

```diff
@@ -0,0 +1,257 @@
+    var queryString = serializeQueryString(allParams);
+    var fullPath = path;
+
+    if (queryString.length > 0) {
+      fullPath += '?' + queryString;
+    }
+
+    const options = {
+      hostname: 'api.trello.com',
+      port: 443,
+      path: fullPath,
+      method: verb
+    };
+
+    const req = https.request(options, (res) => {
+      var dataStr = '';
+      res.on('data', (chunk) => {
+        dataStr += chunk;
+      });
+
+      res.on('end', function () {
+        if (res.statusCode >= 200 && res.statusCode <= 299) {
+          console.log(verb + ' ' + path + ' with ' + JSON.stringify(params) + ' SUCCESS ');
+          resolve(dataStr);
+        } else {
+          console.error(verb + ' ' + path + ' with ' + JSON.stringify(params) + ' FAIL');
+          reject(res.statusCode + ': ' + dataStr);
+        }
+      });
+    });
+
+    req.on('error', (e) => {
+      reject(e);
+    });
+
+    req.end();
+  });
+
+/****************************************\
+ WORKFLOW FUNCTIONS
+\****************************************/
+var copyCard = (ctx, cardID) =>
+  trelloAPICall(ctx, 'POST', '/1/cards/', {
+    idList: ctx.meta.target_list_ID,
+    idCardSource: cardID,
+    pos: 'top',
+    keepFromSource: ''
+  });
+
+var linkCard = (ctx, cardID1, cardID2) =>
+  trelloAPICall(ctx, 'POST', '/1/cards/' + cardID1 + '/attachments/', {
+    url: 'https://trello.com/c/' + cardID2,
+  });
+
+var removeLinkCard = (ctx, cardID1, attachementID) =>
+  trelloAPICall(ctx, 'DELETE', '/1/cards/' + cardID1 + '/attachments/' + attachementID);
+
+// TODO: only create webhook if it doesn't exists
+var webhookCard = (ctx, cardID) =>
+  trelloAPICall(ctx, 'POST', '/1/webhooks/', {
+      callbackURL: ctx.meta.webhook_URL,
+      idModel: cardID
+    });
+
+var removeWebhook = (ctx, webhookID) =>
+  trelloAPICall(ctx, 'DELETE', '/1/webhooks/' + webhookID);
+
+var getWebhooks = (ctx) =>
+  trelloAPICall(ctx, 'GET', '/1/tokens/' + ctx.secrets.api_token + '/webhooks', {});
+
+var getAttachements = (ctx, cardID) =>
+  // get attached cards
+  trelloAPICall(ctx, 'GET', '/1/cards/' + cardID + '/attachments', {
+    fields: 'id,url'
+  });
+
+var closeCard = (ctx, cardID) =>
+  trelloAPICall(ctx, 'PUT', '/1/cards/' + cardID, {
+    closed: true
+  });
+
+var removeWebhooksForModel = async (ctx, modelID) => {
+  try {
+    // get all webhooks
+    const webhooksData = await getWebhooks(ctx);
+    const webhooks = JSON.parse(webhooksData);
+    // filter to keep only relevant webhooks
+    const modelWebhooks = webhooks.filter(webhook => (webhook.idModel === modelID && webhook.callbackURL === ctx.meta.webhook_URL));
+    modelWebhooks.forEach(webhook => {
+      var webhookID = webhook.id;
+      // remove webhook
+      removeWebhook(ctx, webhookID);
+    });
+  }
+  catch (err) {
+    console.error('Could not remove webhooks for modelID ' + modelID);
+    console.error(err);
+  }
+};
+
+var workflowAddMember = async (ctx, cardID) => {
```

Comment:
May be more clear if you use `workflowOnAddMember`, or `workflowAddedMember` so we clearly understand it's the workflow triggered when a member was added.